indexing-handlers/src/test/java/no/unit/nva/indexing/handlers/IndexResourceHandlerTest.java

```diff
@@ -104,6 +116,66 @@ void shouldAddDocumentToIndexWhenResourceExistsInResourcesStorage() throws Excep
         assertThat(allIndexedDocuments, contains(SAMPLE_RESOURCE.resource()));
     }
 
+    @Test
+    void shouldExpandDocumentIfItContainsContributors() throws IOException {
+
+        var json = createResourceJsonWithContributors();
+        var sampleResource =
+                createSampleResourceFromJson(SortableIdentifier.next(), RESOURCES, json);
+        URI resourceLocation = prepareEventStorageResourceFile(sampleResource);
+        InputStream input = createEventBridgeEvent(resourceLocation);
+        indexResourceHandler.handleRequest(input, output, context);
+        var indexedDocument =
+                indexingClient.getIndex(sampleResource.getIndexName()).stream().findFirst().get();
```

Comment:
`.orElseThrow()` instead of `get()`, it removes warning: `'Optional. get()' without 'isPresent()' check `